backend/src/modules/api/api.controller.ts

```diff
@@ -1,13 +1,15 @@
-import { Controller, Get, HttpException, HttpStatus, Logger, Query, Req } from '@nestjs/common';
+import { Controller, Get, HttpException, HttpStatus, Logger, Query, Req, UseGuards } from '@nestjs/common';
 import { ApiService } from './api.service';
 import { RoomEntity } from '../../../db/entities/room.entity';
 import { IGetActiveRoomsRes } from './interfaces';
+import { JwtAuthGuard } from '../auth/jwt-auth.guard';
 
 @Controller()
 export class ApiController {
   private readonly logger = new Logger(ApiService.name);
   constructor(private readonly apiService: ApiService) {}
 
+  @UseGuards(JwtAuthGuard) 
```

Comment:
It is better to use guard for entire controller 

https://docs.nestjs.com/guards
<img width="365" alt="image" src="https://github.com/user-attachments/assets/6c3351d5-d5ec-4197-8947-f2a5e27686b7">